apps/cowswap-frontend/src/utils/orderUtils/getOrderFilledAmount.ts

```diff
@@ -1,10 +1,11 @@
-import { ZERO_BIG_NUMBER } from '@cowprotocol/common-const'
 import { isSellOrder } from '@cowprotocol/common-utils'
 
 import BigNumber from 'bignumber.js'
 
 import { Order } from 'legacy/state/orders/actions'
 
+const ZERO_BIG_NUMBER = new BigNumber(0)
```

Comment:
ubernit: probably there's more places where this const is created. more central place? or overkill?